lib/reporters/cli/index.js

```diff
@@ -81,15 +83,55 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
         // for some reason, if there is no run summary, it is unexpected and hence don't validate this
         var run = this.summary.run;
 
+        // show the body of the response for single request
+        if (options.singleRequest &&
```

Comment:
We need a cleaner way to add branches for `singleRequest` and `run`. Probably separate it out into two different functions and call either of them based on the flag.

Currently we're adding these checks in an ad-hoc manner. Plus, it becomes difficult to add changes to `run` output without affecting `request` output.

Reply:
This change has been incorporated. Thanks!